### PR TITLE
Added basic login and Sign up forms

### PR DESCRIPTION
**Pull request by cheddartot**
basic, unformated HTML pages added with sign up and login forms, values passed to routes.py for use in API later

### CODE REVIEW

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (pgayee694) [inline .idea/inspectionProfiles/profiles_settings.xml]: stuff in ide specific folders (like the .idea folder) should not be commited

File: app/forms.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from flask_wtf import FlaskForm
```
Comment (pgayee694) [inline app/forms.py:1]: add flask_wtf to requirements.txt

File: app/routes.py [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+
+@app.route('/login/', methods=['GET', 'POST'])
+def login():
+    form_log_in = LoginForm()
+    if form_log_in.validate_on_submit():
+        flash('Login requested for user {}, remember_me={}'.format(
+            form_log_in.username.data, form_log_in.remember_me.data))
+        return redirect('/')
+    return render_template('login.html', title='Log In', form=form_log_in)
+@app.route('/signup/', methods=['GET', 'POST'])
```
Comment (pgayee694) [inline app/routes.py:18]: have a blank line between functions

File: app/templates/login.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+    <!DOCTYPE html>
```
Comment (pgayee694) [inline app/templates/login.html]: fix indentation in this file

File: app/templates/signup.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+ <!DOCTYPE html>
```
Comment (pgayee694) [inline app/templates/signup.html:1]: fix indentation in this file

File: app/forms.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+class LoginForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    password = PasswordField('Password', validators=[DataRequired()])
+    remember_me = BooleanField('Remember Me')
+    submit1 = SubmitField('Log In')
+
+class SignInForm(FlaskForm):
+    username = StringField('UsernameSignUp', validators=[DataRequired()])
+    password = PasswordField('PasswordSignUp', validators=[DataRequired()])
```
Comment (pgayee694) [inline app/forms.py]: These field names show up really weird on the the web-page. These could probably be changed to 'Username' and 'Password' respectively

**Review by pgayee694 — CHANGES_REQUESTED** [review]
Looks good, just a few changes

**Review by pgayee694 — APPROVED** [review]

**Review by tkorisko — APPROVED** [review]
All looks good, but add location form on signup.

**Review by bbuffington1662 — APPROVED** [review]

**Pull request merged**